src/main/docs/guide/languageSupport/kotlin/kaptOrKsp.adoc

```diff
@@ -0,0 +1,9 @@
+Micronaut Framework has offered support for Kotlin via https://kotlinlang.org/docs/reference/kapt.html[Kapt].
+
+With version 4.0, Micronaut Framework supports Kotlin also via https://kotlinlang.org/docs/ksp-overview.html[Kotlin Symbol Processing (KSP) API].
+
+Please note that `kapt` is in maintenance mode. Micronaut Kotlin users should move to our KSP support.
+
+____
+kapt is in maintenance mode. We are keeping it up-to-date with recent Kotlin and Java releases but have no plans to implement new features.
```

Comment:
Rephrase to:

```
Please note that KAPT is in maintenance mode. Micronaut 4 includes experimental support for KSP which Kotlin users should consider migrating in the future.
```

Reply:
Close via https://github.com/micronaut-projects/micronaut-core/pull/8786/commits/d788a01765b9f0558b7bf4534797537827f0783e